.github/workflows/test.yml

```diff
@@ -29,15 +34,19 @@ jobs:
       with:
         ruby-version: ${{ matrix.ruby }}
     - name: Install dependencies
+      env: 
+        WITH_LATEST_RELINE: ${{matrix.with_latest_reline}}
```

Comment:
We can use a top level `env` so that every step has access to this environment variable (see [documentation](https://docs.github.com/en/actions/learn-github-actions/environment-variables)).